### ISSUE TITLE
Is transforms.RandomResizedCrop(88) really needed in pretrain stage?

### ISSUE THREAD

**Issue by aiyolo** [yaoyao-liu/meta-transfer-learning#10]
![image](https://user-images.githubusercontent.com/12464091/66712129-9d50b680-edca-11e9-95d5-bf3778fd3de0.png)
The picture above shows the code of data augmentation of the train data in Pretrain stage,  while transforms.RandomResizedCrop(88) operation might lose image information(see the picture below, the third one, others are original, resize(92) respectively), thus has no benetfit for learning.
![image](https://user-images.githubusercontent.com/12464091/66712173-439cbc00-edcb-11e9-8bfc-a56fb982e7a6.png)

**Comment by yaoyao-liu** [comment]
Thanks for your interest in our work.

`transforms.RandomResizedCrop()` is used to solve the overfitting problem when pre-training a deep network (e.g. resnet-25) on a small dataset (e.g. miniImageNet). If  `transforms.RandomResizedCrop()` is not applied, the training accuracy will increase to about 99% very quickly. The network won't be updated after several epochs.

Most of the outputs of `transforms.RandomResizedCrop()` are meaningful. However, there will still be some failure cases like the images you provide. Despite these failure cases, `transforms.RandomResizedCrop()` is still able to improve the performance of the pre-trained model on the validation set.

If you are still not sure about the effectiveness of `transforms.RandomResizedCrop()`, you may remove this operation and re-run the pre-train phase to see the change of validation accuracy.

Besides, using `transforms.RandomResizedCrop()` is very common in training classifiers with deep neural networks. If you have any further questions, feel free to create another issue or add additional comments in this issue.

**Comment by aiyolo** [comment]
Thank you very much!
